docs/configuration-reference.md

```diff
@@ -2037,3 +2040,148 @@ exe:
 cmdline:
   [- <string>]
 ```
+
+### mysqld_exporter_config
+
+The `mysqld_exporter_config` block configures the `mysqld_exporter` integration,
+which is an embedded version of
+[`mysqld_exporter`](https://github.com/prometheus/mysqld_exporter)
+and allows for collection metrics from MySQL servers.
+
+Note that currently, an Agent can only collect metrics from a single MySQL
+server. If you want to collect metrics from multiple servers, run multiple
+Agents and add labels using `relabel_configs` to differentiate between the MySQL
+servers:
+
+```yaml
+mysqld_exporter:
+  enabled: true
+  data_source_name: root@(server-a:3306)/
+  relabel_configs:
+  - source_labels: [__address__]
+    target_label: mysql_server
+    replacement: server-a
+```
+
```

Comment:
nit (non-blocking): Add a description above the full config reference, like we have for other integrations
```suggestion

Full reference of options:

```